cypress/integration/paradigm.spec.js

```diff
@@ -147,12 +147,19 @@ describe('paradigms can be toggled by the show more/less button', () => {
 })
 
 // Can lift skip off this once admin can be tested.
-describe.skip('Paradigm labels', () => {
+describe('Paradigm labels', () => {
   let lemma = 'nipâw'
   let englishLabel = 'they'
   let nehiyawewinLabel = 'wiyanaw'
   let linguisticLabel = '3s'
 
+  beforeEach(() => {
+    // As of 2021-06-02: paradigm label switching is only available to
+    // logged-in users. If you are reading this, and paradigm label switching
+    // is enabled for all users, please delete this entire beforeEach() block.
+    cy.login()
```

Comment:
You could have a test that the menu doesn’t appear when not logged in. Then, when the feature is available to all, that test will start failing, and someone will delete it.